conf/messages.en

```diff
@@ -538,11 +543,11 @@ clientFirstContactPhone.error.invalid = Telephone number can only include number
 clientFirstContactPhone.error.length = The first client contact’s telephone number must be 24 characters or less
 clientFirstContactPhone.change.hidden = Change first client contact telephone number
 
-clientHaveSecondContact.title = Do you have a second client contact if your first contact is not available?
+clientHaveSecondContact.title = Do you have a second contact?
 clientHaveSecondContact.heading = Do you have a second client contact if {0} is not available?
-clientHaveSecondContact.checkYourAnswersLabel = Do you have a second client contact if the first client contact is not available?
+clientHaveSecondContact.checkYourAnswersLabel = Do you have a second contact?
 clientHaveSecondContact.error.required = Select yes if you have a second client contact if the first client contact is not available
-clientHaveSecondContact.change.hidden = Change do you have a second client contact if the first client contact is not available?
+clientHaveSecondContact.change.hidden = Change if you have a second contact
```

Comment:
@AashuChauhanHmrc These changes do not relate to your ticket. They would change one of the pages on the client contact details journey. Please revert them back to what they were.

Reply:
changes related to different story wrongly committed , reverted the commit.

Reply:
@AashuChauhanHmrc you seem to have reverted almost all of your changes including the work that was actually part of this ticket.